app/src/main/java/fi/metatavu/muisti/exhibitionui/persistence/model/Layout.kt

```diff
@@ -26,8 +26,7 @@ data class Layout (
     @TypeConverters(UUIDConverter::class)
     val layoutId: UUID,
 
-    @NonNull
-    val name: String,
+    val name: String?,
```

Comment:
Changing this to be nullable would require migration, so change this back to nullable and use default name for unnamed layouts.